html/videoroomtest.html

```diff
@@ -71,7 +71,7 @@
 					<button class="btn btn-default" autocomplete="off" id="start">Start</button>
 				</h1>
 			</div>
-			<div class="container" id="details">
```

Comment:
Container in container. It is not a good practice. You can check the wrappers on the demo site - they are shifted. Inspect it.